bentoml/_internal/runner/utils.py

```diff
@@ -308,8 +310,8 @@ def _init_var() -> t.Tuple["ctypes.CDLL", t.Dict[str, "_SimpleCData[t.Any]"]]:
         res = _drv.cuInit(0)
         if res != CUDA_SUCCESS:
             _drv.cuGetErrorString(res, ctypes.byref(err))
-            logger.error(f"cuInit failed with error code {res}: {err.value.decode()}")
-        return _drv, plc
+            logger.error(f"cuInit failed with error code {res}: {err.value.decode()}")  # type: ignore
```

Comment:
`ctypes.c_char_p` is `Optional[bytes]`, and theres not really a way to define types vis `ctypes.byref`